### ISSUE TITLE
Example from getting started in README.md fails

### ISSUE THREAD

**Issue by kamil-kaczmarek** [catalyst-team/catalyst#1189]
## 🐛 Bug Report
Running example from the `README.md` fails with `TypeError`.

### How To Reproduce
1. Create new virtualenv, python3.6, catalyst==21.4.1
1. Run example from the `getting started` section in README.md
3. observe TypeError :)

```
Traceback (most recent call last):
  File "main.py", line 45, in <module>
    load_best_on_end=True,
  File "/home/kamil/envs/intcatalyst/lib/python3.6/site-packages/catalyst/runners/runner.py", line 321, in train
    self.run()
  File "/home/kamil/envs/intcatalyst/lib/python3.6/site-packages/catalyst/core/runner.py", line 679, in run
    self._run_event("on_exception")
  File "/home/kamil/envs/intcatalyst/lib/python3.6/site-packages/catalyst/core/runner.py", line 602, in _run_event
    getattr(self, event)(self)
  File "/home/kamil/envs/intcatalyst/lib/python3.6/site-packages/catalyst/core/runner.py", line 594, in on_exception
    raise self.exception
  File "/home/kamil/envs/intcatalyst/lib/python3.6/site-packages/catalyst/core/runner.py", line 676, in run
    self._run_experiment()
  File "/home/kamil/envs/intcatalyst/lib/python3.6/site-packages/catalyst/core/runner.py", line 666, in _run_experiment
    self._run_stage()
  File "/home/kamil/envs/intcatalyst/lib/python3.6/site-packages/catalyst/core/runner.py", line 649, in _run_stage
    self._run_epoch()
  File "/home/kamil/envs/intcatalyst/lib/python3.6/site-packages/catalyst/core/runner.py", line 638, in _run_epoch
    self._run_loader()
  File "/home/kamil/envs/intcatalyst/lib/python3.6/site-packages/catalyst/core/runner.py", line 625, in _run_loader
    self._run_event("on_loader_start")
  File "/home/kamil/envs/intcatalyst/lib/python3.6/site-packages/catalyst/core/runner.py", line 600, in _run_event
    getattr(callback, event)(self)
  File "/home/kamil/envs/intcatalyst/lib/python3.6/site-packages/catalyst/callbacks/misc.py", line 200, in on_loader_start
    desc=f"{runner.stage_epoch_step}/{runner.stage_epoch_len}"
TypeError: __init__() got an unexpected keyword argument 'desc'

```

#### Code sample
It was copied from `README.md`, master branch.

**Comment by Scitator** [comment]
Could you please provide more details? (we have a special script for that).
Current CI works great with 3.6/3.7/3.8 python and 1.3+ PyTorch. Moreover, I have run a quick sanity check and could not reproduce the error - https://colab.research.google.com/drive/1-n9FMN2D35hawNn4PeAjlbmkZenBDIvd?usp=sharing

**Comment by kamil-kaczmarek** [comment]
Here is an output from `catalyst-contrib collect-env`

```
Collecting environment information...
Catalyst version: 21.04.1
PyTorch version: 1.8.1+cu102
Is debug build: No
CUDA used to build PyTorch: 10.2
TensorFlow version: N/A
TensorBoard version: N/A

OS: Ubuntu 18.04.5 LTS
GCC version: (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
CMake version: Could not collect

Python version: 3.6
Is CUDA available: No
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA

Versions of relevant libraries:
[pip3] catalyst==21.4.1
[pip3] numpy==1.19.5
[pip3] tensorboardX==2.2
[pip3] torch==1.8.1
[conda] Could not collect
```

Hope this helps.

**Comment by kamil-kaczmarek** [comment]
I did a bit of research on this.

Problem disappears when I installed tqdm-4.60.0 (in the Google Colab it's already in the env).

**Comment by Scitator** [comment]
🤔  looks like ``torch.tqdm`` somehow depends on just tqdm, good catch! we will return the tqdm dependency in such a case :)

**Comment by kamil-kaczmarek** [comment]
Sounds good thanks!

**Comment by kamil-kaczmarek** [comment]
fixed in #1192
